src/realm/array_mixed.cpp

```diff
@@ -151,6 +151,14 @@ Mixed ArrayMixed::get(size_t ndx) const
                             ObjKey(m_int_pairs.get(payload_ndx + 1))};
                 return Mixed(ret);
             }
+            case type_UUID: {
+                ensure_string_array();
+                REALM_ASSERT(size_t(int_val) < m_strings.size());
+                auto s = m_strings.get(payload_ndx);
+                UUID id;
+                memcpy(&id, s.data(), sizeof(UUID));
```

Comment:
It could be problematic to have `sizeof(UUID)` be part of the file format. Maybe use `UUIDBytes` here to make it more robust?

Reply:
Thanks for the benchmark; I will update.